src/docs/product/dashboards/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Dashboards
+sidebar_order: 6
+redirect_from:
+  - /workflow/visibility/
+  - /workflow/dashboards/
+  - /product/error-monitoring/dashboards/
+description: "Sentry's Dashboards help you gain detailed perspectives of your application's health by providing filtering tools to navigate your issues across multiple projects in a single view."
```

Comment:
```suggestion
description: "Sentry's Dashboards provide a single view of your application's health, including the ability to navigate issues across multiple projects."
```

Reply:
Should this be `issues and performance` instead of only issues?

Reply:
I've updated this to become:

> Dashboards provide a single view of your application's health, including the ability to navigate issues and performance across multiple projects.